### ISSUE TITLE
sylius_currency_code_choice doesn't exist

### ISSUE THREAD

**Issue by criolit** [Sylius/Sylius#2469]
Hi,

When I am trying to create an order via API I am getting:

```
{
"code": 500,
"message": "Could not load type \"sylius_currency_code_choice\"",
"errors": null
}
```

I've did search for "sylius_currency_code_choice" and I just found

```
->add('currency', 'sylius_currency_code_choice', array(
        'constraints' => array(
        new NotBlank()
    )
))
```

here

    src/Sylius/Bundle/CoreBundle/Form/Type/Api/OrderType.php

No other matches.

**Comment by Tetragramat** [comment]
In \Sylius\Bundle\CurrencyBundle\Form\Type\CurrencyChoiceType is 'sylius_currency_choice'

If you remove 'code_' from 'sylius_currency_code_choice', then it work.

**Comment by michalmarcinkowski** [comment]
Fixed in #5896.
